test/nano_80x_cfg.py

```diff
@@ -48,7 +48,7 @@
     outputCommands = process.NanoAODEDMEventContent.outputCommands,
     compressionLevel = cms.untracked.int32(9),
     compressionAlgorithm = cms.untracked.string("LZMA"),
-
+    fakeName =cms.untracked.bool(True),
```

Comment:
this should have been fakeNameForCrab ... fixing it (apologies but I was coding from the phone while boarding )